AppOne/AppOne/DataServiceLayer/ServiceLayer.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ServiceLayer.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 13.12.22.
+//
+
+import Foundation
+
+protocol ServiceCalls {
```

Comment:
Not a good naming. Conventions suggests something with "able" or "ing" for the ending